torchok/constructor/constructor.py

```diff
@@ -291,12 +291,27 @@ def create_dataloaders(self, phase: Phase) -> List[DataLoader]:
             return []
 
     @staticmethod
-    def _prepare_dataloader(dataset_params: DictConfig, dataloader_params: DictConfig) -> DataLoader:
+    def _prepare_sampler(dataset: ImageDataset, sampler_params: DictConfig) -> Sampler:
```

Comment:
This method does't need. In our philosophy if you have parameters in config for some class you can not add additional parameters inside the code. So the sampler must be create only with options like:
```
sampler = SAMPLERS.get(sampler_params.name)(**sampler_params.params)
```
This philosophy makes torchok more simplify to understand, and code become more readable.
Also not each Sampler has parameter for weights. So we wouldn't have ability to add new Samplers.

For create sampler like in code above, you need to create own Sampler Weighted class which inherit from torch Sampler. And it's parameters should be weights_csv_path and weights_column for take the weights.